server/src/test/java/io/druid/client/CachingClusteredClientFunctionalityTest.java

```diff
@@ -0,0 +1,227 @@
+/*
+ * Druid - a distributed column store.
```

Comment:
Oh, cool.  Ok, I just grabbed the license from one of the old files and it was still the old thing.  Hadn't been keeping up on the language, will update to this.